tensorboard/examples/plugins/example_raw_scalars/tensorboard_plugin_example_raw_scalars/plugin.py

```diff
@@ -35,6 +35,7 @@
 from tensorboard.plugins import base_plugin
 from tensorboard.util import tensor_util
 from tensorboard.plugins.scalar import metadata
+from util import can_serve_from_static
```

Comment:
This doesn’t actually work for me: `util` is not the fully qualified
name of a module reachable from `sys.path`, and this is specified as an
absolute import, not a relative import. When creating a new Python 3.7
virtualenv, installing `tf-nightly` into it, running `setup.py develop`,
and then launching TensorBoard, I see:

```
  File ".../tensorboard/tensorboard/examples/plugins/example_raw_scalars/tensorboard_plugin_example_raw_scalars/plugin.py", line 38, in <module>
    from util import can_serve_from_static
ModuleNotFoundError: No module named 'util'
```

The reason that the corresponding import works in the test is that when
the test is run as a main module, its working directory is inserted into
`sys.path`.

Can you elaborate on how you installed and ran the example plugin in
your testing? I’m not sure what environment this could have worked in;
this module has `from __future__ import absolute_import` (as it should),
so the semantics should be the same even in Python 2.

In any case, [relative imports are forbidden by Google style][1],
precisely so that we don’t have to think about these sorts of things. So
this import, if it exists at all, needs to be:

```python
from tensorboard_plugin_example_raw_scalars import util
```

(note furthermore that we only import modules, not their exports).

[1]: https://google.github.io/styleguide/pyguide.html#224-decision